Automattic/wp-calypso
# SectionHeader: Add Learn more support link in PeopleListSectionHeader

#### Proposed Changes

* Iteration over https://github.com/Automattic/wp-calypso/pull/71474
* Add Learn more support link in `PeopleListSectionHeader` by: 
  * Refactor `SectionHeader` by removing `popoverText` prop
  * Refactor `PeopleListSectionHeader` to pass `SupportInfo` using the label prop.
  * Add `privacyLink` false to avoid showing the privacy link on the Followers page.
  * Add Learn more `link` to https://wordpress.com/support/followers/

#### Testing Instructions



1. Visit `Users` > `Followers` and ensure the popover doesn't show "Privacy Information" link. Instead, we display a `Learn more` link.
2. Click on the link and observe a new page is open to the localized support page https://wordpress.com/support/followers/
3. Click on `Email Subscribers` and verify the Learn more link is visible and working.

#### Screenshots

**Followers**

| **Before** | **After** |
|---|---|
| <img width="614" alt="Screenshot 2022-12-29 at 12 48 52" src="https://user-images.githubusercontent.com/779993/209947272-a53d6bbb-188c-4cea-bc35-db0e0ec1ecd7.png"> |<img width="641" alt="Screenshot 2022-12-29 at 12 25 38" src="https://user-images.githubusercontent.com/779993/209944789-5bc6eedb-b105-41d8-bd73-c7621eb84c5b.png"> |

**Email Subscribers**

| **Before** | **After** |
|---|---|
| <img width="640" alt="Screenshot 2022-12-29 at 12 50 51" src="https://user-images.githubusercontent.com/779993/209947232-ff262444-d429-42d6-b462-bc1cb8ac2268.png"> | <img width="677" alt="Screenshot 2022-12-29 at 12 25 51" src="https://user-images.githubusercontent.com/779993/209944787-ceddc2ab-6780-46d6-86b9-ec9330113610.png"> |


#### Pre-merge Checklist





- Closes https://github.com/Automattic/wp-calypso/issues/69947
